packages/admin/cms-admin/src/dependencies/createDependencyMethods.ts

```diff
@@ -29,8 +39,16 @@ export function createDependencyMethods<RootBlocks extends Record<string, BlockI
                         node: ${rootQueryName}(id: $id) {
                             id
                             ${Object.keys(rootBlocks).join("\n")}
+                            ${
+                                scopeFragment
+                                    ? `scope {
```

Comment:
As discussed yesterday, the assumption that the field is called scope all the time is probably right, but technically incorrect.